source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern allows you to create complex
+immutable objects separately from their representation. The
+:doc:`builders </fundamentals/builders/>`
+provided by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to:
+
+* Use the builder pattern to specify sort criteria for your queries.
+* Learn how to perform ascending sorts, descending sorts, and combining sorts.
+* Sort on the text score of :manual:`$text </reference/operator/query/text/>` 
+  queries.
+
+If you are new to sorting in the
+MongoDB Java driver, consider reading our
+:doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`
+for a general introduction to sorting with the MongoDB Java driver.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all
+sort criteria operators supported by MongoDB. Each method returns
+an instance of the
+:java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
+type. 
+
+.. _docs_java_fundamentals_builders_sort_sort_example:
```

Comment:
Nitpick: I don't think we need extremely descriptive ref anchors, and am fairly certain that they're all local to the repository now. Search this repository for examples.

Reply:
Thanks for the note Chris. I have shortened the ref anchor here: https://github.com/mongodb/docs-java/pull/59/commits/845251e43a4a068ce54fad366fc1e7f0376f3150

Going forward, do you know how I could have determined for myself that ref anchors were local to the repository?